_posts/2021-01-07-cockpit-235.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Cockpit 235
+author: pitti
+date: 2021-01-07
+tags: cockpit linux
+slug: cockpit-235
+category: release
+summary: Cockpit with SSH key improvements and motd editing
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 235.
+
+## Login: Improved handling of SSH host keys
+
+When connecting to a secondary host directly from the login page, unknown and changed host keys are handled in a better way. The situation is explained and accepted keys are remembered in the Browser:
+
+![login page SSH fingerprint](/images/login-fingerprint.png)
+
+## Overview: Editable motd
+
+The displayed motd now has an edit button where you can change `etc/motd`:
```

Comment:
```suggestion
The Overview page's motd now has an edit button to change `/etc/motd`.

*[motd]: message of the day
```

1. I removed the colon here too.
2. Specified Overview page.
3. Dropped "where you can".
4. I _think_ I'm using the footnote correctly here. It should show up as a dotted underline which defines the term both in screen readers and on also on hovers in browser.